src/Msg.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
```

Comment:
I have a pull request for the filewriter that drastically simplifies this file. Maybe we can use a bit if that code here? Lets talk about it later.